vim25/types/helpers.go

```diff
@@ -316,6 +322,42 @@ func (ci VirtualMachineConfigInfo) ToConfigSpec() VirtualMachineConfigSpec {
 	return cs
 }
 
+type ByteArray []byte
```

Comment:
I also wonder if we should not just create a file `byte_array.go` in case we have more of these in the future? It would make finding them much easier...

Reply:
Done, to byte_slice.go